jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/BuildImageStep.java

```diff
@@ -185,4 +187,47 @@ public DescriptorDigest getDiffId() throws LayerPropertyNotFoundException {
       return imageBuilder.build();
     }
   }
+
+  @Nullable
+  private ImmutableList<String> computeEntrypoint(
+      Image<Layer> baseImage, ContainerConfiguration containerConfiguration) {
+    boolean shouldInheritEntrypoint = containerConfiguration.getEntrypoint() == null;
```

Comment:
This is not consistent with program arguments in that it doesn't inherit if it is an empty list. Reason?

Reply:
Tomcat have no entrypoing but cmd. I changed how to deal with it. Tell me if it suits you now.